### PR TITLE
Less Folders

### PR DESCRIPTION
**Pull request by qiqvo**
Removed folders, search functions. 

Added `instance_name` to every class, so now object literally knows its name. And we are able not to send it to get-request functions.

Some classes (like new and the most important Launch-class) has references to other classes, ie:
launch = [.., 'rocket',...]
with new init func in Base we can specify and init with a copy:
`self._inside['rocket'] = Rocket(copy=raw['rocket'])`
look at `launch.__parse` func.

### CODE REVIEW

**Comment by veetaw** [comment]
ok I didn't see this PR, ignore my previous comments lol

I'll take a look

File: llpy/agency.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from llpy import LLpyException, Base
+from llpy.constants import URL
+
+
+class Agency(Base):
+    instance_name = ['agency', 'agencies']
+    def __get_the(self, query):
+        return Base.__get_the(self, query)
```
Comment (veetaw) [inline llpy/agency.py:8]: you can't to this as far as I know :/

**Comment by veetaw** [comment]
there are issues with private methods, I'll fix them tomorrow if I'll have some free time

File: llpy/agency.py (hunk 1 above)
Comment (qiqvo) [reply llpy/agency.py:8]: test case:
```
class A:
   field = 'aaaa'
   def pr(self):
       print(self.field)

class B(A):
   field = 'bbbb'

def main():
   a = A()
   b = B()
   a.pr()
   b.pr()
```

if you don't mean initing 'instance_name', then I didn't get the thing. :)

Comment (veetaw) [reply llpy/agency.py:8]: I mean, getting private methods from another class with this syntax.

**Comment by veetaw** [comment]
I'm going to merge and fix some errors, thanks

**Comment by veetaw** [comment]
I think your idea is nice but not well implemented

all commits will be on https://github.com/veetaw/llpy/tree/inherritance

**Pull request merged**